src/main/scala/nl/knaw/dans/easy/stage/Conf.scala

```diff
@@ -7,14 +7,15 @@ import org.rogach.scallop.{singleArgConverter, ValueConverter, ScallopConf}
 
 class Conf(args: Seq[String]) extends ScallopConf(args) {
   printedName = "easy-stage-dataset"
-  version(s"$printedName ${Version()}")
+  val indent_____ = printedName.replaceAll(".", " ")
```

Comment:
The underscores ensure that the indenting below looks nice both in code and when using the `--help` switch on the command line.